time-series-forecast/resource/evaluate.R

```diff
@@ -145,13 +145,13 @@ ComputeErrorMetricsCrossval <- function(crossvalDfList, rollingWindow = 1.0) {
     w <- as.integer(rollingWindow * nrow(tmpDf))
     w <- max(w, 1)
     w <- min(w, nrow(tmpDf))
-    # ME and MPE are not implemented in Prophet
-    tmpDf[["ME"]] <- prophet:::rolling_mean(tmpDf$y - tmpDf$yhat, w)
-    tmpDf[["MPE"]] <- prophet:::rolling_mean((tmpDf$y - tmpDf$yhat)/tmpDf$y, w)
-    # Other error metrics have built-in prophet implementations
-    tmpDf[["MAE"]] <- prophet:::mae(tmpDf, w)
-    tmpDf[["MAPE"]] <- prophet:::mape(tmpDf, w)
-    tmpDf[["RMSE"]] <- prophet:::rmse(tmpDf, w)
+    # # ME and MPE are not implemented in Prophet
```

Comment:
as per my previous comment, let's copy the functions from prophet without importing it and uncomment these lines without prophet::: (which was dirty anyway)

Reply:
This is done at the top of the script!